README.adoc

```diff
@@ -1,43 +1,175 @@
+  -v, --verbose                       More output.
+----
+
+=== `binding`
+
+----
+Configure and manage a Kamelet binding.
+
+Usage:
+  kn-source-kamelet binding [command]
+
+Examples:
+
+  # Configure and manage a Kamelet binding.
+  kn-source-kamelet binding create|update|delete
+
+Available Commands:
+  create      Create Kamelet bindings and bind source to Knative broker, channel or service.
+
+Flags:
+  -h, --help   help for binding
+
+Use "kn-source-kamelet binding [command] --help" for more information about a command.
+----
+
+==== `binding create`
+
+----
+Create Kamelet bindings and bind source to Knative broker, channel or service.
+
+Usage:
+  kn-source-kamelet binding create [flags]
+
+Examples:
+
+  # Create Kamelet binding with source and sink.
+  kn-source-kamelet binding create NAME
+
+  # Add a binding properties
+  kn-source-kamelet binding create NAME --kamelet=name --sink|broker|channel|service=<name> --source-property=<key>=<value>
+
+Flags:
+      --broker string                 Uses a broker as binding sink.
+      --channel string                Uses a channel as binding sink.
+  -h, --help                          help for create
+      --force bool                    Apply the changes even if the binding already exists.
+      --kamelet string                Kamelet source.
+  -n, --namespace string              Specify the namespace to operate in.
+      --service string                Uses a Knative service as binding sink.
+      --sink string                   Sink expression to define the binding sink.
+      --source-property stringArray   Add a source property in the form of "<key>=<value>"
+----
+
+=== `bind`
+
+Shortcut version of `kn-source-kamelet binding create` with Kamelet source as positional argument.
+The shortcut command auto generates a binding name in case no explicit name is given as command option `--name`.
+
+----
+Create Kamelet bindings and bind source to Knative broker, channel or service.
+
+Usage:
+  kn-source-kamelet bind [flags]
+
+Examples:
+
+  # Bind Kamelets to a Knative sink
+  kn-source-kamelet bind SOURCE
+
+  # Add a binding properties
+  kn-source-kamelet bind SOURCE --sink|broker|channel|service=<name> --source-property=<key>=<value>
 
 Flags:
-  -h, --help   help for list
+      --broker string                 Uses a broker as binding sink.
+      --channel string                Uses a channel as binding sink.
+  -h, --help                          help for bind
+      --force bool                    Apply the changes even if the binding already exists.
+      --name string                   Binding name.
+  -n, --namespace string              Specify the namespace to operate in.
+      --service string                Uses a Knative service as binding sink.
+      --sink string                   Sink expression to define the binding sink.
+      --source-property stringArray   Add a source property in the form of "<key>=<value>"
```

Comment:
After you have remove `--sink-property`, would it make sense to just rename it to `--property` ? ("source" is kind of self-evident because its already part of the command name)

Would make it a bit shorter, too.

Reply:
Same for `create binding` probably.

Reply:
good point